### PR TITLE
Merged Trevor changes

### PR DESCRIPTION
**Pull request by cacraig**
- Merged in changes from https://github.com/trevorcolenps/ekip-api:master (https://github.com/18F/ekip-api/pull/228).
- Kept the deployment script as-is, and cleared up a conflict.

### CODE REVIEW

File: ekip/nationalparks/data/pass-list-20150828.csv [hunk 1]
```diff
@@ -177,7 +177,7 @@ Modoc NF - Warner Mountain District,530-279-6116,Cedarville,CA,http://www.fs.fed
 Mojave National Preserve,760-252-6108,Kelso,CA,http://www.nps.gov/moja/index.htm,YES,YES,YES
 "Muir Woods National Monument (see also Golden Gate NRA, CA)",415-388-7368,Mill Valley,CA,http://www.nps.gov/muwo,YES,YES,YES
 Needles BLM Field Office,760-326-7000,Needles,CA,http://www.blm.gov/ca/st/en/fo/needles.html,YES,YES,YES
-New Melones Lake - Reclamation Central California Area Office,209-536-9094,Sonora,CA,http://www.usbr.gov/mp/ccao/newmelones/index.html ��,YES,YES,YES
+New Melones Lake - Reclamation Central California Area Office,209-536-9094,Sonora,CA,http://www.usbr.gov/mp/ccao/newmelones/index.html Â ,YES,YES,YES
```
Comment (khandelwal) [inline ekip/nationalparks/data/pass-list-20150828.csv]: There are some encoding related issues here. Can we ensure that they don't affect anything?

**Comment by cacraig** [comment]
@khandelwal I merged in Trevor's changes, and made a few conflict fixes, and reverted the cf.sh script to what was there previously. If it looks good to you, we can merge this branch, and then close #228

File: ekip/nationalparks/data/pass-list-20150828.csv (hunk 1 above)
Comment (cacraig) [inline ekip/nationalparks/data/pass-list-20150828.csv]: Good catch. I can fix this real quick.

Comment (cacraig) [inline ekip/nationalparks/data/pass-list-20150828.csv]: Although, this file isn't necessary anymore due to the recent redemption list changes.

**Pull request merged**